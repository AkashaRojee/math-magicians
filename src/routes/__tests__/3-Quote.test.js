import renderer from 'react-test-renderer';
import Quote from '../3-Quote';

test('Quote route renders correctly', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});