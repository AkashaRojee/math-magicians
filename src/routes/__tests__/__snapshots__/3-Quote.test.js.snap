// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote route renders correctly 1`] = `
<main
  className="quote-route"
>
  <div>
    <p>
      Mathematics is not about numbers, equations, computations or algorithms: it is about understanding. - William Paul Thurston
    </p>
  </div>
</main>
`;
